### DISCUSSION TITLE
Checking in. Is this project still active?

### DISCUSSION THREAD

**Discussion by birds-swim** [hakavlad/prelockd#15] [Q&A]
Hello! 

I'm using Debian 12 Bookworm. I was researching OOM programs for my home PC and I found your project in the process. Looks great! Very interesting. 

I wanted to know if using prelockd would be a good fit/compatible with Debian 12 Stable. I am currently using the Linux 6.9 kernel from the Backports as of this writing.

In your "How-to Configure", you say: 

> Edit the config (/etc/prelockd.conf or /usr/local/etc/prelockd.conf) and restart the service.

But I'm not sure how to tweak this file.

Also, do you accept donations? I'd really love to tip my Dev and say thank you for this cool software.

**Comment by Michael679089** [comment]
How about Earlyoom? They have similar goals of preventing Out of Memory moments

**Reply by hakavlad** [reply]
earltyoom does not protect page cache. Enable MRGLRU to protect.

**Comment by hakavlad** [comment]
Use (enable) MGLRU instead of prelockd.
